vue.config.js

```diff
@@ -9,6 +9,15 @@ const defineGlobals = (config) => {
     args[0] = defined;
     return args;
   });
+  config.module
+    .rule('babel')
+    .test(/base-scraper\.js$/)
```

Comment:
What is it?

Did you forget to remove it?

`base-scraper` is from `israeli-bank-scrapers`

Reply:
Thats actually an interesting one. The built version of israeli bank scrapers in the typescript branch has a class with a property on it which fails the build so this was targeted at that, but I could add this to all js files.

Reply:
But adding this to all js files makes the build much longer. I think a better solution is to update electron which will have a more up to date node version and won't need this. But I don't want to mix an electron update in this pr.